response-datastore/response-server-service/src/main/java/com/google/cloud/healthcare/fdamystudies/config/AppConfig.java

```diff
@@ -8,7 +8,15 @@
 
 package com.google.cloud.healthcare.fdamystudies.config;
 
+import com.google.cloud.GcpLaunchStage.Beta;
```

Comment:
Please update file copyright to `2020-2021`

Reply:
Fixed comment.